src/DataModelRegistry.hpp

```diff
@@ -56,10 +56,19 @@ class NODE_EDITOR_PUBLIC DataModelRegistry
   using RegisteredModelsMap =
           std::unordered_map<QString, RegistryItemPtr>;
 
+  DataModelRegistry() = default;
+  ~DataModelRegistry() = default;
+
+  DataModelRegistry(const DataModelRegistry&) = delete;
+  DataModelRegistry(DataModelRegistry&&) = default;
+
+  DataModelRegistry& operator=(const DataModelRegistry&) = delete;
```

Comment:
It is a minor code style thing:
instead of 
`function_name(const Type &)`
I write everywhere
`function_name(Type const &)`